packages/tasks/src/tasks/video-classification/inference.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+/**
+ * Inputs for Video Classification inference
+ */
+export interface VideoClassificationInput {
+	/**
+	 * The input video data
+	 */
+	data: unknown;
+	/**
+	 * Additional inference parameters
+	 */
+	parameters?: VideoClassificationParameters;
+	[property: string]: unknown;
+}
+/**
+ * Additional inference parameters
+ *
+ * Additional inference parameters for Video Classification
+ */
+export interface VideoClassificationParameters {
+	/**
+	 * The sampling rate used to select frames from the video.
+	 */
+	frameSamplingRate?: number;
+	/**
+	 * The function to apply to the model outputs in order to retrieve the scores.
+	 */
+	functionToApply?: TextClassificationOutputTransform;
+	/**
+	 * The number of sampled frames to consider for classification.
+	 */
+	numFrames?: number;
+	/**
+	 * When specified, limits the output to the top K most probable classes.
+	 */
+	topK?: number;
+	[property: string]: unknown;
+}
+export type TextClassificationOutputTransform = "sigmoid" | "softmax" | "none";
```

Comment:
Maybe let's rename to a more general class (`ClassificationOutputTransform`?)

Reply:
Yes - let's move the classification output to `schema-utils.json`